numpy/core/src/common/simd/sse/arithmetic.h

```diff
@@ -91,6 +91,58 @@ NPY_FINLINE __m128i npyv_mul_u8(__m128i a, __m128i b)
 // TODO: emulate integer division
 #define npyv_div_f32 _mm_div_ps
 #define npyv_div_f64 _mm_div_pd
+
+// Ref: Agner Fog's <instrset.h>, XXX This code is common for all of SIMD, where to put?
```

Comment:
you can put them here
https://github.com/numpy/numpy/blob/f5f845b7bd0b737740e419826698ab4a9e61564d/numpy/core/src/common/simd/simd_utils.h#L2-L3